test/frontend/EventList.test.js

```diff
@@ -50,23 +50,21 @@ describe('<EventList />', () => {
       ' array',
     async () => {
       returnEventArray();
-      const wrapper = await mount(<EventList />);
+      const wrapper = await mount(<EventList {...RENDERED_EVENTS} />);
       wrapper.update();
-      expect(wrapper.find(EventCard)).to.have.lengthOf(
+      expect(wrapper.find(EventList)).to.have.lengthOf(
         RENDERED_EVENTS.responseData.length
       );
     }
   );
   it('Should render a title if no events are returned', async () => {
     returnEmptyArray();
-    const wrapper = await mount(<EventList />);
+    const wrapper = await mount(<EventList {...RENDERED_EVENTS} />);
+    const result = wrapper.find('.no-event-list');
```

Comment:
this isn't testing whether it would render 'no events yet'. you can add this code block to the previous test and it would pass